File: fibonacci_series/fibonacci_series.py
import sys

def main(file_arg):
    with open(file_arg) as f:
        for line in f:
            number = int(line)
            print(fibo(number))

def fibo(number):
    if number == 1 or number == 2:
        return(1)
    elif number == 0:
        return(0)
    else:
        return(fibo(number - 2) + fibo(number-1))

if __name__ == '__main__':
    main(sys.argv[1])

File: fibonacci_series/fibonacci_series.rb
filename = ARGV[0]

def fibo(number)
  if number == 1 or number == 2
    1
  elsif number == 0
    0
  else
    fibo(number - 2) + fibo(number - 1)
  end
end

f = File.open(filename, "r") do |file|
  file.each_line do |line|
    number = line.to_i
	puts(fibo(number))
  end
end
